src/node/game.js

```diff
@@ -1,30 +1,49 @@
 const Board = require('./board.js');
-const AllPlayers = require('./allPlayers');
 const TurnCounter = require('./turnCounter.js');
+const Referee = require('./referee.js');
+const Player = require('./player.js');
+const constants = require('./constants.js');
+const gameStatus = require('./gameStatus.js');
 
 function Game() {
   const game = {
+    numOfPlays: 0,
     board: Board(),
-    players: AllPlayers(),
-    turnTracker: TurnCounter(),
+    players: constructArrayOfPlayers(),
+    turnCounter: TurnCounter(),
+    referee: Referee(),
     start: () => runTheGame(game),
+    playerWhoHasTheTurn: turn => {
+      return game.players[turn];
+    },
   };
   return game;
 }
 
 function runTheGame(game) {
-  while (game.players.hasNoWinner()) {
+  while (game.referee.getGameStatus() === gameStatus.ONGOING) {
```

Comment:
You could've declared this function, inside the Game function! that way you would have access to the game instance, and would avoid the need of passing `game` as variable.
That is, unless you want to actually run the game via this function (which is fine). Otherwise, running the game like:
```
const game = Game();
game.start()
```
Should be more of the pattern you are used to.

TL;DR;
To be more function programming like, have 2 types of structure, DATA and DATA MANIPULATORS.
Game would be only a simple object and every function in this file would receive receive some DATA and return new  value of DATA.
To be more object oriented programming like, create object with behaviors. Expose abstractions (like .start method) and hide implementation details (for example, playerWhoHasTheTurn don't need to receive `turn` as parameter if instance of game can get it by itself via game.turnCounter.getTurn()). Methods needs less parameters, because everything should be already present in the instance!